tests/python/python_test_sparsify_fps.py

```diff
@@ -79,7 +79,8 @@ def test_fps_restart(self):
         # I first select 50
         r_fps = fps(self.x, 50, 0, method="simple")
 
-        # I ask for 100 but provide the restart tuple. Selection will continue from point 51
+        # I ask for 100 but provide the restart tuple.
+        # Selection will continue from point 51
```

Comment:
That comment almost reads like a `GOTO 51` statement...

Reply:
tell Michele about that

Reply:
This is a completely unrelated Python file that got touched by autopep8; I suggest we ignore it